examples/dataset/keypoint_detection/keypoint_detection/benchmark_model.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from argparse import ArgumentParser
+from argparse import Namespace
+
+import pandas as pd
+from keypoint_detection.metrics import compute_metrics
+from keypoint_detection.model import infer_random
+from keypoint_detection.model import infer_retinaface
+from tqdm import tqdm
+
+import kolena
+from kolena._experimental.dataset import fetch_dataset
+from kolena._experimental.dataset import test
+
+
+def run(args: Namespace) -> int:
+    kolena.initialize(verbose=True)
+    infer = infer_retinaface if args.model == "RetinaFace" else infer_random
+    df = fetch_dataset(args.dataset)
+
+    inferences = []
+    for record in tqdm(df.itertuples(), total=len(df)):
```

Comment:
Do we want to be using [progress bar](https://github.com/kolenaIO/kolena/blob/trunk/kolena/_utils/log.py#L83) instead to give consistent UI?

Reply:
Since `progress_bar` is internal (`kolena._utils`) it's a tad smelly to import it in these example scripts. Opted for vanilla `tqdm` accordingly